runconfig/opts/parse.go

```diff
@@ -246,7 +246,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*container.Config, *container.Host
 	// Validate if the given hostname is RFC 1123 (https://tools.ietf.org/html/rfc1123) compliant.
 	hostname := *flHostname
 	if hostname != "" {
-		// Linux hostname is limited to HOST_NAME_MAX=64, not not including the terminating null byte.
+		// Linux hostname is limited to HOST_NAME_MAX=64, not to include the terminating null byte.
```

Comment:
i think the problem was just a duplication of `not`, so this should just be `not including`